src/devices/Ws2812B/BitmapImageNeo3.cs

```diff
@@ -0,0 +1,57 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Iot.Device.Bindings.Utils;
+
+namespace Iot.Device.Bindings.WS2812B
+{
+    /// <summary>
+    /// Special 24bit RGB format for Neo pixel LEDs where each bit is converted to 3 bits.
+    /// A one is converted to 110, a zero is converted to 100.
+    /// </summary>
+    public class BitmapImageNeo3 : BitmapImage
+    {
+        const int BytesPerComponent = 3;
+        const int BytesPerPixel = BytesPerComponent * 3;
+        const int ResetDelayInBytes = 30; // 100us @ 2.4Mbps
```

Comment:
please use time units instead (TimeSpan or miliseconds) unless datasheet defines it like that then please use actual SPI speed.

Reply:
btw this feels like should be in the device itself and not in the bitmap

Reply:
The bitmap format is local to the binding, so I consider it already part of the device. Would it be better to make it a private class? I feel it is a bit messy to add this constant to the constructor of the bitmap.
As for the speed, I'm not sure what you mean. Freq should always be 2.4Mbps for these LEDs. Do you mean I should calculate this constant from say `internal const int Ws2812b.SpiSpeed = 2_400_000`? What if someone add a BitmapImageNeo1 (1Bpp, requires extra HW)? Then I think it is better to move the Spi freq to BitmapImageNeo3, since that is what is dictating the freq.

Reply:
Let's do it private for now and leave the delay as is. Once we figure out the common interface I suspect we will need to refactor a lot of similar classes all over the place